.github/workflows/ci-test.yml

```diff
@@ -274,6 +274,7 @@ jobs:
           exit $num_fails
       - name: "Send Message"
         uses: ./.github/actions/slack-file
+        if: ${{ always() }}
```

Comment:
when do we plan to switch to trigger on failure()?

Reply:
It initially was failure() and we didn't see any leading to suspicions.  I think at some point I'll have to force a failure in a test in a pr to see that the step would trigger even though it wouldn't have access to secrets to push a message?   Or maybe work in another branch.   Guess I should byte the bullet and do that soon.

Reply:
>  I think at some point I'll have to force a failure in a test in a pr to see that the step would trigger even though it wouldn't have access to secrets to push a message?

Sounds like a good canary plan.  I think it is sufficient to verify the step gets triggered with the expected message content.  It doesn't need the secrets to actually push.